packages/rollup-plugin-html/rollup-plugin-html.js

```diff
@@ -24,6 +24,14 @@ const { createHtmlAsset, createHtmlAssets } = require('./src/createHtmlAssets');
 const watchMode = process.env.ROLLUP_WATCH === 'true';
 const defaultFileName = 'index.html';
 
+/**
+ * @param {string} id
+ * @param {string | undefined} rootDir
+ */
+function isAbsoluteUrl(id, rootDir) {
+  return rootDir && id.startsWith('/') && !id.startsWith(rootDir);
```

Comment:
You already check for the presence of `rootDir` below, so you can make it a required parameter here.